app/global-error.tsx

```
@@ -0,0 +1,36 @@
+      <body>
+        <div role="alert" className="p-4 border border-red-300 rounded">
+          <h2 className="font-semibold">Something went wrong</h2>
+          <p>Please try again or reload the page.</p>
+          <details className="mt-2 text-sm text-gray-600">
+            <summary>Error details</summary>
+            <pre className="mt-1 whitespace-pre-wrap">{error.message}</pre>
+          </details>
+          <button
+            className="mt-2 underline mr-4"
+            onClick={() => reset()}
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)